CHANGELOG.md

```diff
@@ -4,15 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-### unreleased
+### 0.12.0
 
-**Released**: tbd
+**Released**: 2021.04.01
 
-**Commit Delta**: [Change from 0.11.0 release](https://github.com/plus3it/tardigrade-ci/compare/0.11.0...unreleased)
+**Commit Delta**: [Change from 0.11.0 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.0...unreleased)
 
 **Summary**:
 
-*   Pins terraform-docs version using multi-stage docker build
```

Comment:
It would have been fine if my code changes had been reviewed before Juno's changes.  I submitted my PR last night before he even moved his ticket to "Ready ".  I can correct it.

Reply:
That comment was in place last week. Juno's current PR hasn't been merged yet. But it might be. And then we'd have to adjust this PR. Totally normal, nothing to get excited about.